Badge Count content iterations (#1094)
hashicorp/design-system: website/docs/components/badge-count/partials/code/how-to-use.md

## How to use this component

> The default invocation requires a text argument. By default, a medium neutral filled Badge Count is generated.

Reviewer: ```suggestion
The default invocation requires a text argument and renders a medium neutral filled Badge Count.
```
Could potentially remove the duplicative "defaults"
